src/runtime/runtime_k210.go

```diff
@@ -0,0 +1,159 @@
+// +build k210
+
+// This file implements target-specific things for the K210 chip as used in the
+// MAix Bit with Mic.
+
+package runtime
+
+import (
+	"device/kendryte"
+	"device/riscv"
+	"machine"
+	"runtime/volatile"
+	"unsafe"
+)
+
+type timeUnit int64
+
+func postinit() {}
+
+//export main
+func main() {
+
+	// Both harts should disable all interrupts on startup.
+	initPLIC()
+
+	// Only use one hart for the moment.
+	if riscv.MHARTID.Get() == 0 {
```

Comment:
Perhaps you can reformat this code to be more Go style, to reserve indentation for exceptions.

```go
	if riscv.MHARTID.Get() != 0 {
		abort()
	}
	// normal code
```

Reply:
done